client.js

```diff
@@ -103,21 +102,44 @@ function createReporter() {
     overlay = require('./client-overlay');
   }
 
-
+  var styles = {
```

Comment:
This stuff looks quite nice, but should probably have been in a separate PR.

Might as well leave it in now you've done it though.